### ISSUE TITLE
Test_integrity_sync System test failed with present event

### ISSUE THREAD

**Issue by Deblintrake09** [wazuh/wazuh-qa#4425]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.5.1-rc1 | Authd tests | Manager | Packages/Sources | Linux |

After launching the system tests for 4.5.1-rc1, it was found that the `test_cluster/test_integrity_sync/test_integrity_sync.py::test_zip_size_limit` test  fails because the log on `worker2`was not detected, but the log is present. 

- Timeout error shown:
   ```
   E    TimeoutError: Did not found the expected callback in wazuh-worker2: .*Files to create: 0 \| Files to update: 0 \| Files to delete: 0.*
   ```

- Logs in agent:

   ```
   2023/08/15 01:56:41 INFO: [Worker wazuh-worker2] [Integrity sync] Files to create: 0 | Files to update: 0 | Files to delete: 0
   2023/08/15 01:56:41 INFO: [Worker wazuh-worker2] [Integrity sync] Finished in 0.002s.
   2023/08/15 01:56:45 INFO: [Worker wazuh-worker2] [Agent-info sync] Starting.
   2023/08/15 01:56:45 INFO: [Worker wazuh-worker2] [Agent-info sync] Finished in 0.002s. Updated 0 chunks.
   2023/08/15 01:56:48 INFO: [Worker wazuh-worker2] [Agent-groups recv] Starting.
   2023/08/15 01:56:48 INFO: [Worker wazuh-worker2] [Agent-groups recv] Finished in 0.003s. Updated 1 chunks.
   2023/08/15 01:56:50 INFO: [Worker wazuh-worker2] [Integrity check] Starting.
   2023/08/15 01:56:50 INFO: [Worker wazuh-worker2] [Integrity check] Finished in 0.053s. Sync required.
   2023/08/15 01:56:50 INFO: [Worker wazuh-worker2] [Integrity sync] Starting.
   2023/08/15 01:56:50 INFO: [Worker wazuh-worker2] [Integrity sync] Files to create: 0 | Files to update: 0 | Files to delete: 0
   2023/08/15 01:56:50 INFO: [Worker wazuh-worker2] [Integrity sync] Finished in 0.002s.
   2023/08/15 01:56:55 INFO: [Worker wazuh-worker2] [Agent-info sync] Starting.
   2023/08/15 01:56:55 INFO: [Worker wazuh-worker2] [Agent-info sync] Finished in 0.002s. Updated 0 chunks.
   2023/08/15 01:56:58 INFO: [Worker wazuh-worker2] [Agent-groups recv] Starting.
   2023/08/15 01:56:58 INFO: [Worker wazuh-worker2] [Agent-groups recv] Finished in 0.003s. Updated 1 chunks.
   2023/08/15 01:56:59 INFO: [Worker wazuh-worker2] [Integrity check] Starting.
   2023/08/15 01:56:59 INFO: [Worker wazuh-worker2] [Integrity check] Finished in 0.054s. Sync required.
   2023/08/15 01:56:59 INFO: [Worker wazuh-worker2] [Integrity sync] Starting.
   2023/08/15 01:56:59 INFO: [Worker wazuh-worker2] [Integrity sync] Files to create: 0 | Files to update: 0 | Files to delete: 0
   2023/08/15 01:56:59 INFO: [Worker wazuh-worker2] [Integrity sync] Finished in 0.002s.
   ```

**Evidence:** [agentless_cluster.zip](https://github.com/wazuh/wazuh/files/12343622/agentless_cluster.zip)

**Comment by Rebits** [comment]
Probably fixed with https://github.com/wazuh/wazuh-qa/issues/4368

**Comment by Rebits** [comment]
Issue fixed in 4.6.0 https://github.com/wazuh/wazuh-qa/issues/4368. 
Closing this issue
